terracommon/tropp/serializers.py

```diff
@@ -25,11 +25,11 @@ def get_attribute(self, instance):
 
 class SimpleViewpointSerializer(serializers.ModelSerializer):
     picture = SerializerMethodField()
-    geometry = GeometryField(source='point.geom', read_only=True)
+    point = GeometryField(source='point.geom')
```

Comment:
Why read_only is dropped here ?

Reply:
`SimpleViewpointSerializer` is only used in `DetailCampaignNestedSerializer` and on list action on viewpoints